lib/coins/utils.js

```diff
@@ -8,7 +8,7 @@ const coins = {
   ETH: {
     displayScale: 15,
     unitScale: 18,
-    zeroConf: false
+    zeroConf: true
```

Comment:
Where is thing being used?

Reply:
On line 3400:

`return coin.zeroConf && tx.fiat.gte(this.trader.zeroConfLimits[tx.cryptoCode])`